src/test/java/org/takes/facets/hamcrest/HmRsTextBodyTest.java

```diff
@@ -0,0 +1,61 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2014-2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.takes.facets.hamcrest;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+import org.takes.rs.RsWithBody;
+
+/**
+ * Test case for {@link HmRsTextBody}.
+ *
+ * @author Izbassar Tolegen (t.izbassar@gmail.com)
+ * @version $Id$
+ * @since 2.0
+ */
+public final class HmRsTextBodyTest {
+
+    /**
+     * HmRsTextBody can test if body contains text.
+     */
+    @Test
+    public void testsBodyValueContainsText() {
+        MatcherAssert.assertThat(
+            new RsWithBody("<html><h1>Hello</h1></html>"),
+            new HmRsTextBody("<h1>Hello</h1>")
```

Comment:
@t-izbassar this test is not so obvious, please look at previous comment about default matcher in constructor